### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by sausax** [pelme/htpy#43] [General]
I was playing around with the new fasthtml project but multiple people mentioned this htpy while comparing it to fasthtml. That's how I found it. I am so glad that I found it. It is so elegant and beautiful compared to what's out there. I am having so much fun using it my projects. Thank you for creating this.

**Comment by pelme** [comment]
Thanks for the kind words, I am glad it is useful to you! Please open an issue if you figure out something we can improve. If you do something that is open source, feel free to share a link to your project!

**Comment by geigerzaehler** [comment]
Same from me. I have been looking for a library like this for a while and none of the alternatives fit me needs and requirements. (My issues with other libraries were: no escaping, awkward syntax, based on mutations, complex with too many APIs). Thanks for this project
